ledger/consensual/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'corda-api.common-library'
+    id 'corda.common-publishing'
+    id 'org.jetbrains.kotlin.jvm'
+}
+
+description 'Corda Consensual Ledger'
+
+dependencies {
+    api platform(project(':corda-api'))
+
+    api 'net.corda.kotlin:kotlin-stdlib-jdk8-osgi'
+    api 'javax.persistence:javax.persistence-api'
+    api 'org.slf4j:slf4j-api'
+
+    compileOnly 'org.osgi:osgi.annotation'
+    // We only need Hibernate for its annotations.
+    compileOnly("org.hibernate:hibernate-osgi:$hibernateVersion") {
+        exclude group: 'org.osgi'
+    }
```

Comment:
Can this be removed?
I realise it was in the old API, but hopefully in the new one we can avoid a hibernate dependency and only use `javax.persistence` annotations.

Reply:
I have removed it. (And the same from common/)